src/styles/transitions.js

```diff
@@ -33,6 +33,10 @@ const durationInternal = {
   leavingScreen: 195,
 };
 
+const formatMs = (miliseconds) => `${Math.round(miliseconds)}ms`;
+const isString = (value) => typeof value === 'string';
+const isNumber = (value) => !isNaN(parseFloat(value));
```

Comment:
Sounds good to me http://stackoverflow.com/questions/9716468/is-there-any-function-like-isnumeric-in-javascript-to-validate-numbers.